examples/digital_fingerprinting/production/Dockerfile

```diff
@@ -13,35 +13,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG MORPHEUS_CONTAINER=nvcr.io/nvidia/morpheus/morpheus
-ARG MORPHEUS_CONTAINER_VERSION=v24.10.00-runtime
+ARG BASE_IMG=nvcr.io/nvidia/cuda
+ARG BASE_IMG_TAG=12.5.1-base-ubuntu22.04
 
-FROM ${MORPHEUS_CONTAINER}:${MORPHEUS_CONTAINER_VERSION} as base
+FROM ${BASE_IMG}:${BASE_IMG_TAG} as base
 
-# # Fix the entrypoint to work with different WORKDIR
-ENTRYPOINT [ "/opt/conda/bin/tini", "--", "/workspace/docker/entrypoint.sh" ]
+# Install necessary dependencies using apt-get
+RUN apt-get update && apt-get install -y \
+      git \
+      git-lfs \
+      wget \
+    && apt-get clean
 
-SHELL ["/bin/bash", "-c"]
+# Install miniconda
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh \
+    && bash /tmp/miniconda.sh -b -p /opt/conda \
+    && rm /tmp/miniconda.sh
+
+# Add conda to the PATH
+ENV PATH=/opt/conda/bin:$PATH
 
-# Set the workdir to the DFP base folder
-WORKDIR /workspace/examples/digital_fingerprinting/
+# Create base environment
+RUN conda create -y --name morpheus-dfp python=3.10
 
-# Install DFP dependencies
-RUN source activate morpheus \
-    && /opt/conda/bin/conda env update --solver=libmamba -n morpheus --file /workspace/conda/environments/examples_cuda-125_arch-x86_64.yaml
+SHELL ["/bin/bash", "-c"]
 
 # Set the tracking URI for mlflow
 ENV MLFLOW_TRACKING_URI="http://mlflow:5000"
 
 # This will get used by pipelines for the --s3_cache option
 # ENV DFP_S3_CACHE="/work/examples/dfp_workflow/morpheus/.s3_cache"
 
-# If any changes have been made from the base image, recopy the sources
-COPY . /workspace/examples/digital_fingerprinting/
-
 # Set the workdir to be the production folder
 WORKDIR /workspace/examples/digital_fingerprinting/production
 
+# If any changes have been made from the base image, recopy the sources
+COPY . /workspace/examples/digital_fingerprinting/production/
+
+# Install morpheus-dfp and any additional dependencies needed to run the examples
+RUN source activate morpheus-dfp && /opt/conda/bin/conda env update --solver=libmamba -n morpheus-dfp --file ./conda/environments/dfp_example_cuda-125_arch-x86_64.yaml
```

Comment:
I would merge this in with the `conda create` command on line 37 it should be possible to do:
```bash
conda env create --solver=libmamba -y --name morpheus-dfp --file ./conda/environments/dfp_example_cuda-125_arch-x86_64.yaml
```

Reply:
this feels so obvious, now!